packages/frameworks/frameworks.json

```diff
@@ -797,6 +797,34 @@
       }
     }
   },
+  {
+    "name": "Create Elm App",
+    "slug": "create-elm-app",
+    "demo": "https://create-elm-app.now-examples.now.sh",
+    "logo": "https://raw.githubusercontent.com/vercel/vercel/master/packages/frameworks/logos/elm.svg",
+    "tagline": "Create Elm apps with no build configuration.",
+    "description": "An Elm app, bootstrapped with create-elm-app.",
+    "website": "https://github.com/halfzebra/create-elm-app",
+    "detectors": {
+      "every": [
+        {
+          "path": "package.json",
+          "matchContent": "\"(dev)?(d|D)ependencies\":\\s*{[^}]*\"create-elm-app\":\\s*\".+?\"[^}]*}"
+        }
+      ]
+    },
+    "settings": {
+      "buildCommand": {
+        "placeholder": "`npm run build` or `elm-app build`"
+      },
+      "devCommand": {
+        "value": "elm-app start"
```

Comment:
What about `PORT`?

I don't believe there's a way to pass it via command line and `create-elm-app` reads the `PORT` environment variable implicitly.

https://github.com/halfzebra/create-elm-app/blob/f09a3f7183712646eb5ba0a983136b5283d3656c/scripts/start.js#L150

Reply:
If the elm cli can detect the `PORT` environment variable, then this will work as-is 👍